jaxrs/src/main/java/io/narayana/lra/filter/ServerLRAFilter.java

```diff
@@ -95,6 +98,10 @@ public class ServerLRAFilter implements ContainerRequestFilter, ContainerRespons
     @Inject
     LRAParticipantData data;
 
+    @Inject
+    @ConfigProperty(name = ENLIST_PARTICIPANT_CLIENT_MAX_RETRY, defaultValue = "3")
```

Comment:
@arjantijms I am considering the default value here and I think that setting it to 3 would be a major change. WDYT @mmusgrov ?

Reply:
It's good to think about defaults indeed. What would you propose for an alternative? I did consider "1" and "0" here too.

Reply:
Zero is what we currently have. If the lock is held then the probability is that the coordinator is overloaded so telling the client to retry the op will shift the load back to one of the clients whereas retrying at the coordinator will maintain the load on the coordinator (typically there will be one coordinator but many clients). Since the new property is configurable the deployment can decide what value is suitable for its environment.

Reply:
Ah wait, applolgies, this is the client (ServerLRA) sorry. A default of 3 sounds reasonable but 0 is fine too - ie I see pros and cons either way so I will defer to you on this one.